### ISSUE TITLE
Squiz.Operators.IncrementDecrementUsage.Found no longer works with missing spaces

### ISSUE THREAD

**Issue by cgocast** [PHPCSStandards/PHP_CodeSniffer#325]
## Describe the bug
In phpcs 2.5.0, we correctly add a `Squiz.Operators.IncrementDecrementUsage.Found` error when writing `$a+=1` without spaces. In phpcs 3.7.2, the error is missing.

### Code sample
```php
$a=0;
$a += 1; // We correctly get a Squiz.Operators.IncrementDecrementUsage.Found
$a+=1;   // We are missing a Squiz.Operators.IncrementDecrementUsage.Found
echo $a;
```

### Custom ruleset
```xml
<?xml version="1.0"?>
<ruleset name="My Custom Standard">
  <description>If you are using a custom ruleset, please enter the relevant part here.</description>
  <rule ref="Squiz.Operators.IncrementDecrementUsage"/>
</ruleset>
```

### To reproduce
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See error message displayed
```
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 4 | ERROR | Increment operators should be used where possible; found
   |       | "$a += 1;" but expected "++$a"
   |       | (Squiz.Operators.IncrementDecrementUsage.Found)
----------------------------------------------------------------------
```

## Expected behavior
We should have also have an error on line 5
```
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 4 | ERROR | Increment operators should be used where possible; found
   |       | "$a += 1;" but expected "++$a"
   |       | (Squiz.Operators.IncrementDecrementUsage.Found)
 5 | ERROR | Increment operators should be used where possible; found
   |       | "$a+=1;" but expected "++$a"
   |       | (Squiz.Operators.IncrementDecrementUsage.Found)
----------------------------------------------------------------------
```

## Versions (please complete the following information)

| | |
|-|-|
| Operating System | Windows 11
| PHP version | 8.2.13
| PHP_CodeSniffer version | master
| Standard | Squiz
| Install type | Composer (local)

## Please confirm:

- [X] I have searched the issue list and am not opening a duplicate issue.
- [X] I confirm that this bug is a bug in PHP_CodeSniffer and not in one of the external standards.
- [X] I have verified the issue still exists in the `master` branch of PHP_CodeSniffer.

**Comment by jrfnl** [comment]
> In phpcs 2.5.0, we correctly add a Squiz.Operators.IncrementDecrementUsage.Found error when writing $a+=1 without spaces. In phpcs 3.7.2, the error is missing.

I agree with you that the sniff should disregard spaces and that the current behaviour is a bug, but it's a bug which has always existed, so I don't see how you could get different results on PHPCS 2.5.0 - I mean, I've tested the code sample against PHPCS 2.5.0 and the results are exactly the same as with PHPCS 3.7.2 or even `master`.

**Comment by jrfnl** [comment]
PR #329 should fix this. Testing appreciated.

**Comment by cgocast** [comment]
Thank you very much @jrfnl !

I tested branch [feature/325-squiz-incrementdecrementusage-bugfix](https://github.com/PHPCSStandards/PHP_CodeSniffer/tree/feature/325-squiz-incrementdecrementusage-bugfix) and everything works as expected on my side.

Concerning versoin 2.5.0, I noticed I did not use a standard version. Some former employee at my office forked it and fixed the bug in the process. Your bugfix is much better than what he did.

**Comment by jrfnl** [comment]
Thanks for testing @cgocast ! And good to hear the explanation about what happened with your 2.5.0 version. I honestly couldn't fathom how it could have worked in 2.5.0. Maybe next time a colleague is tempted to make a fix, they could submit a PR instead ;-)
